src/main/kotlin/com/github/aivancioglo/resttest/http/Response.kt

```diff
@@ -41,54 +46,81 @@ abstract class Response() {
     }
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
      *
-     * @param verifiers for responseSpecification validation.
+     * @param verifiers for response validation.
      */
     @SafeVarargs
     fun assertThat(verifier: Verifier, vararg verifiers: Verifier) = printFailuresIfExist(verifier, *verifiers)
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
+     *
+     * @param verifiers for response validation.
+     */
+    @SafeVarargs
+    fun assertThat(verifiers: Array<Verifier>) = printFailuresIfExist(*verifiers)
+
+    /**
+     * Making response validation.
      *
-     * @param code of responseSpecification.
-     * @param verifiers for responseSpecification validation.
+     * @param code of response.
```

Comment:
@param code Status code of response.

You can say that it is integer or numeric.